### PR TITLE
update ontology metadata with a comment

### PR DESCRIPTION
**Pull request by SepidehAlassi**
resolves [DSP-610](https://dasch.myjetbrains.com/youtrack/agiles/110-4/111-52?issue=DSP-610)

- [x] change existing comment in the ontology metadata or add a new one + unit tests
- [x] e2e test
- [x] test data
- [x] update documentation

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala [hunk 1]
```diff
@@ -1816,6 +1816,12 @@ class OntologyResponderV2(responderData: ResponderData) extends Responder(respon
                 // Check that the ontology exists and has not been updated by another user since the client last read its metadata.
                 _ <- checkOntologyLastModificationDateBeforeUpdate(internalOntologyIri = internalOntologyIri, expectedLastModificationDate = changeOntologyMetadataRequest.lastModificationDate)
 
+                // get the metadata of the ontology.
+                existingOntologyMetadata: Option[OntologyMetadataV2] <- loadOntologyMetadata(internalOntologyIri)
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala:1820]: You don't want to load the ontology metadata from the triplestore here. It's already in the cache. For performance reasons, ontologies are loaded when Knora starts.

Comment (benjamingeer) [reply webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala:1820]: See the ontology responder design doc here: https://docs.knora.org/05-internals/design/api-v2/ontology-management/

File: webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala [hunk 2]
```diff
@@ -1833,10 +1841,19 @@ class OntologyResponderV2(responderData: ResponderData) extends Responder(respon
 
                 // Check that the update was successful. To do this, we have to undo the SPARQL-escaping of the input.
 
+                // Is there any new label given?
+                label = if (changeOntologyMetadataRequest.label.isEmpty) {
+                    // No. Consider the old label for checking the update.
+                    oldMetadata.label
+                } else
+                    // Yes. Consider the new label for checking the update.
+                    changeOntologyMetadataRequest.label
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala:1850]: Please use curly brackets for both branches of the `if` statement.

**Review by benjamingeer — CHANGES_REQUESTED** [review]
Just some small things to change, otherwise looks good, thanks!

File: webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala (hunk 1 above)
Comment (SepidehAlassi) [reply webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala]: Thank you for pointing this out. I just fixed this.

**Review by benjamingeer — APPROVED** [review]
Great, thank you!

**Pull request merged**